README.md

```diff
@@ -10,12 +10,29 @@ Using flattened Docker images from <https://github.com/u-root/sidecore-images>,
 Building on top of [`cpu`](https://github.com/u-root/cpu), **sidecore** merely
 requires your IoT system to run a small daemon in order to use it as a resource.
 
-## Building
+## Build and run
 
 **NOTE**: Go version 1.20 is required as a minimum.
 
-Run `go build ./cmds/sidecore` to build the command.
+### Git repository
 
-## Usage
+If you have cloned this repository, run `go build ./cmds/sidecore` to build the
+command, then `./sidecore -h` for help.
 
-After building the command, run `./sidecore -h` for help.
+### Quick start
+
+To try out the command without explicitly cloning, run:
+
+```sh
+go run github.com/u-root/sidecore/cmds/sidecore@latest
+```
+
+### Installation
+
+To build the binary into your local Go bin directory, run:
+
+```sh
+go install github.com/u-root/sidecore/cmds/sidecore@latest
+```
+
+Ensure to have that bin directory in your `$PATH` to run it conveniently.
```

Comment:
do you want to tell them what that is, for people unused to go, i.e. say 
To build the binary into your local Go bin directory (usually ~/go/bin), run:

Reply:
I feel like all of the above is already a partial copy of upstream documentation... I'll add a link to that.

Reply:
It's also why I prefer the more common git clone way, which people are probably used to from other projects and is less implicit magic.